src/App.tsx

```diff
@@ -1,25 +1,57 @@
-import React from 'react';
+// import React, { useState } from 'react';
+import { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
+// import { event } from 'cypress/types/jquery';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [lengthItem, setLengthItem] = useState('5');
```

Comment:
here as well. it is basically items per page. so we can use it as is itemsPerPage